{{cookiecutter.project_slug}}/_/deployment/python/deployment/application/base/cronjob/jobs/hello/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../../template
+nameSuffix: -hello
+patchesJson6902:
+- target:
+    group: batch
+    version: v1beta1
+    kind: CronJob
+    name: cron-examples
```

Comment:
This will show up as part of the job name later. Let's use a proper name here, e.g. `{{ project_slug }}-job` or `application-job`.

Reply:
Good idea.  I will go with "application" because using the project slug makes the final name potentially really long, eg. "name-of-the-project-job-backup-production"